src/test/java/org/springframework/integration/dsl/test/file/FileTests.java

```diff
@@ -307,7 +309,11 @@ public IntegrationFlow tailFlow() {
 		@Bean
 		public IntegrationFlow fileReadingFlow() {
 			return IntegrationFlows
-					.from(s -> s.file(tmpDir.getRoot()).patternFilter("*.sitest"),
+					.from(s -> s.file(tmpDir.getRoot())
+									.patternFilter("*.sitest")
+									.useWatchService(true)
```

Comment:
OMG!
Well, since our CI is on the Linux, I guess we can live with that slow OS X honor.